src/main/resources/ledger.mixins.json

```diff
@@ -1,47 +1,62 @@
 {
-    "required": true,
-    "package": "com.github.quiltservertools.ledger.mixin",
-    "mixins": [
-        "AreaHelperMixin",
-        "AxeItemMixin",
-        "BlockItemMixin",
-        "BlockMixin",
-        "DoubleInventoryMixin",
-        "EndCrystalEntityMixin",
-        "ExplosionMixin",
-        "FallingBlockEntityMixin",
-        "FlintAndSteelItemMixin",
-        "FrostWalkerEnchantmentMixin",
-        "HoeItemMixin",
-        "HoneycombItemMixin",
-        "ItemScattererMixin",
-        "LivingEntityMixin",
-        "LockableContainerBlockEntityMixin",
-        "RavagerEntityMixin",
-        "ScreenHandlerMixin",
-        "ShovelItemMixin",
-        "SlotMixin",
-        "SnowGolemEntityMixin",
-        "blocks.AbstractPlantPartBlockMixin",
-        "blocks.AbstractPlantStemBlockMixin",
-        "blocks.BambooBlockMixin",
-        "blocks.DaylightDetectorBlockMixin",
-        "blocks.FarmlandBlockMixin",
-        "blocks.FireBlockMixin",
-        "blocks.FlowerPotBlockMixin",
-        "blocks.FluidBlockMixin",
-        "blocks.IceBlockMixin",
-        "blocks.LeavesBlockMixin",
-        "blocks.LeverBlockMixin",
-        "blocks.NoteBlockMixin",
-        "blocks.RootedDirtBlockMixin",
-        "blocks.SpongeBlockMixin",
-        "blocks.coral.CoralBlockBlockMixin",
-        "blocks.coral.CoralBlockMixin",
-        "blocks.coral.CoralFanBlockMixin",
-        "blocks.coral.CoralWallFanBlockMixin"
-    ],
-    "injectors": {
-        "defaultRequire": 1
+  "required": true,
```

Comment:
Please set the IDE formatter to 4 spaces per tab

Reply:
should be fixed, had json defaulting to 2 spaces.